ui/pages/add-collectible/add-collectible.js

```diff
@@ -46,8 +50,66 @@ export default function AddCollectible() {
   );
   const [tokenId, setTokenId] = useState('');
   const [disabled, setDisabled] = useState(true);
+  const [eventTracked, setEventTracked] = useState([
+    {
+      event: 'Token Added',
+      address: '',
+      tracked: false,
+    },
+  ]);
+
   const [collectibleAddFailed, setCollectibleAddFailed] = useState(false);
 
+  const trackEvent = useContext(MetaMetricsContext);
+  const collectibles = useSelector(getCollectibles);
+
+  useEffect(() => {
+    if (!tokenId || !address) {
+      return;
+    }
+
+    const newCollectible = collectibles.find(
+      (collectible) =>
+        collectible.address === address &&
```

Comment:
might be worth making this a case-insensitive check until we have thoroughly standardized checksumming across controllers.

Reply:
fixed [here](https://github.com/MetaMask/metamask-extension/pull/14279/commits/d68dc9564e5ee8b6f7989ccbbb799b3604389567)